bpf/bpf_lxc.c

```diff
@@ -539,11 +530,7 @@ static __always_inline int handle_ipv6_from_lxc(struct __ctx_buff *ctx, __u32 *d
 	if (ipv6_store_flowlabel(ctx, ETH_HLEN, SECLABEL_NB) < 0)
 		return DROP_WRITE_ERROR;
 
-#ifdef ENABLE_WIREGUARD
-	if (dst_remote_ep)
-		set_encrypt_mark(ctx);
-	else
-#elif !defined(TUNNEL_MODE)
+#if !defined(TUNNEL_MODE)
```

Comment:
Nit:
```suggestion
#ifndef TUNNEL_MODE
```
+ `/* TUNNEL_MODE */` comment after the `#endif`?